cms/config/api.ts

```diff
@@ -1,7 +1,7 @@
 export default {
   rest: {
     defaultLimit: 25,
-    maxLimit: 100,
+    maxLimit: 9999,
```

Comment:
Because of the previous value, you'll only see the first 100 countries in the combobox if you use the staging API.